tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -694,73 +695,65 @@ public int compare(T o1, T o2) {
+     * Set the baseline (or "trunk") value and status for this VoteResolver.
      *
-     * Assume that we don't need to make any changes for INHERITANCE_MARKER here;
-     * the input will have INHERITANCE_MARKER if appropriate; do nothing special
-     * for a specific value that happens to match the Bailey value.
-     *
-     * Reference: https://unicode.org/cldr/trac/ticket/11857
-     *
-     * @param trunkValue the trunk value
-     * @param trunkStatus the trunk status
-     *
-     * TODO: consider renaming: setTrunk to setBaseline; getTrunkValue to getBaselineValue; getTrunkStatus to getBaselineStatus
+     * @param baselineValue the baseline value
+     * @param baselineStatus the baseline status
      */
-    public void setTrunk(T trunkValue, Status trunkStatus) {
-        this.trunkValue = trunkValue;
-        this.trunkStatus = trunkValue == null ? Status.missing : trunkStatus;
+    public void setBaseline(T baselineValue, Status baselineStatus) {
+        this.baselineValue = baselineValue;
+        this.baselineStatus = baselineValue == null ? Status.missing : baselineStatus;
     }
 
-    public T getTrunkValue() {
-        return trunkValue;
+    public T getBaselineValue() {
+        return baselineValue;
     }
 
-    public Status getTrunkStatus() {
-        return trunkStatus;
+    public Status getBaselineStatus() {
+        return baselineStatus;
     }
 
     /**
-     * Set the number of required votes for this VoteResolver, based on the given locale and PathHeader
+     * Set the locale and PathHeader for this VoteResolver
      *
      * You must call this whenever you are using a VoteResolver with a new locale or a new PathHeader
      *
      * @param locale the CLDRLocale
      * @param pathHeader the PathHeader
      */
     public void setLocale(CLDRLocale locale, PathHeader pathHeader) {
-        requiredVotes = supplementalDataInfo.getRequiredVotes(locale, pathHeader);
+        this.locale = locale;
+        this.pathHeader = pathHeader;
     }
 
     /**
      * What are the required votes for this item?
+     *
      * @return the number of votes (as of this writing: usually 4, 8 for established locales)
      */
     public int getRequiredVotes() {
-        return requiredVotes;
-    }
-
-    /**
-     * What are the adjusted required votes for this item?
-     *
-     * @return requiredVotes, or LOWER_BAR if appropriate
-     */
-    public int getAdjustedRequiredVotes() {
-        if (requiredVotes == HIGH_BAR && trunkStatus != Status.approved) {
-            return LOWER_BAR;
-        } else {
-            return requiredVotes;
+        if (requiredVotes == 0) {
+            int preliminaryRequiredVotes = supplementalDataInfo.getRequiredVotes(locale, pathHeader);
+            if (preliminaryRequiredVotes == HIGH_BAR && baselineStatus != Status.approved) {
+                requiredVotes = LOWER_BAR;
+            } else {
+                requiredVotes = preliminaryRequiredVotes;
+            }
```

Comment:
All the "adjustment" is now encapsulated here; this is refactoring, doesn't change behavior -- this is cleaner, avoids awkwardly beginning the calculation of required votes in setLocale before the VoteResolver is fully initialized